charts/cosigned/values.schema.json

```diff
@@ -75,7 +76,8 @@
                     "secretKeyRef": {
                         "name": ""
                     },
-                    "cosignPub": ""
+                    "cosignPub": "",
+                    "webhookName": ""
```

Comment:
Shouldn't it be `webhookName=cosigned.sigstore.dev` now ?

Reply:
Done.

Reply:
Probably, I will need to rebase after #128